CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.1.0
```

Comment:
Can we follow the same logic of the last release by adding the commit and the PR #?

Reply:
+1, that would be an improvement.
I am good with approving the PR as that shouldn't block the PR IMO

Reply:
Done

Reply:
Thank you!